### DISCUSSION TITLE
wallet balance

### DISCUSSION THREAD

**Discussion by p30hashemi** [Chia-Network/chia-blockchain#11213] [Support]
hi supporter

Shows the wallet balance after syncing incorrectly.
What should I do to have the correct display?

**Comment by loppefaaret** [comment]
Without knowing what OS you are on, i would ask you to go into your users home folder, under `~/.chia/mainnet/wallet/db/` and manually move the current wallet database out, while the Chia client is closed down!
After you have manually moved wallet databse out, open up the Chia client again, and let it resync a fresh wallet database from 0
